scipy/signal/filter_design.py

```diff
@@ -3504,6 +3504,196 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, ab=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    Parameters
+    ----------
+    w0 : float
+        A scalar that must satisfy  0 < w0 < 1. For notch filters,
+        w0 is the normalized frequency to remove from the filtered
+        signal (w0 = 1 corresponds to half of the sample frequency).
+    bw : float
+        Filter bandwidth.
```

Comment:
Could it be explained better and in more detail? Maybe mention that at bandwidth edge frequencies the attenuation is determined by `ab`.